### PR TITLE
rm fork-choice param

### PR DESCRIPTION
**Pull request by tonatoz**

### CODE REVIEW

File: chains.yaml [hunk 1]
```diff
@@ -118,6 +113,13 @@ chain-settings:
           grpcId: 1002
           chain-id: 0x89
           short-names: [polygon, matic]
+        - id: Mumbai
```
Comment (a10zn8) [inline chains.yaml:116]: resurrected

File: chains.yaml [hunk 2]
```diff
@@ -206,14 +205,13 @@ chain-settings:
       label: Polygon zkEVM
       type: eth
       settings:
-        options:
-          validate-peers: false
         expected-block-time: 2.7s
         allow-pruning-requirement: true
+        options:
+          disable-validation: true
```
Comment (a10zn8) [inline chains.yaml:211]: resurrected?

File: chains.yaml [hunk 3]
```diff
@@ -233,14 +231,13 @@ chain-settings:
       label: Arbitrum Nova
       type: eth
       settings:
-        options:
-          validate-peers: false
         expected-block-time: 260ms
         allow-pruning-requirement: true
+        options:
+          disable-validation: true
```
Comment (a10zn8) [inline chains.yaml:237]: resurrected?

File: chains.yaml [hunk 4]
```diff
@@ -355,10 +346,10 @@ chain-settings:
         allow-pruning-requirement: true
         options:
           validate-peers: false
+          validate-syncing: false
```
Comment (a10zn8) [inline chains.yaml:349]: resurrected?

File: chains.yaml [hunk 5]
```diff
@@ -400,14 +390,13 @@ chain-settings:
       label: Mantle
       type: eth
       settings:
-        expected-block-time: 2s
+        expected-block-time: 500ms
```
Comment (a10zn8) [inline chains.yaml:393]: resurrected?

**Review by a10zn8 — CHANGES_REQUESTED** [review]
a lot of revert

**Comment by tonatoz** [comment]
> a lot of revert

fixed

**Review by a10zn8 — APPROVED** [review]

**Pull request merged**